test/resttests.ts

```diff
@@ -128,9 +128,9 @@ describe('Rest Tests', function () {
     it('gets a non-existant resource (404)', async() => {
         try {
             let restRes: restm.IRestResponse<HttpBinData> = await _rest.get<HttpBinData>('https://httpbin.org/status/404');
-            
+
             assert(restRes.statusCode == 404, "statusCode should be 404");
-            assert(restRes.result == null, "object should be null");
+            assert(restRes.result === null, "object should be null");
```

Comment:
result object should be null?

Reply:
According to your documentation in https://github.com/Microsoft/typed-rest-client/blob/master/README.md a 404 should return `null`.

Reply:
Yep - in the REST layer since it's job is to return a restful object, a 404 should return a null.  However the http layer will return body.

Reply:
I think I'm fine with these lines / changes

Reply:
I did a poor job of writing that comment. I meant that the string could be changed to be more expressive.